orchestra/todos/serializers.py

```diff
@@ -174,3 +123,20 @@ class Meta:
             'additional_data')
         read_only_fields = ('id',)
         list_serializer_class = TodoBulkCreateListSerializer
+
+
+class BulkTodoSerializerWithQAField(BulkTodoSerializer):
+    qa = serializers.SerializerMethodField()
+
+    def get_qa(self, obj):
+        return None
+
+    class Meta(BulkTodoSerializer.Meta):
+        fields = BulkTodoSerializer.Meta.fields + ('qa',)
+
+
+class BulkTodoSerializerWithQASerializer(BulkTodoSerializer):
```

Comment:
i don't understand the difference between this serializer and the one above it

Reply:
`BulkTodoSerializerWithQAField` replaces `TodoSerializer` and its `qa` field always returns `None` :)

Reply:
... while this (`BulkTodoSerializerWithQASerializer`) one's `qa` is a `TodoQASerializer` record